db/migrate/20231107235725_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Good job so far creating the model for posts, but it would be nice if you could add references for the author that will create author_id and a corresponding index as required. Let's fix that with the code below 👍🏽

👇🏾
`t.references :author, foreign_key: { to_table: 'users' }`

Reply:
I believe you required changes have to bee reviewed. The reason is that when running the migration code for the project the following error appears:
```sh
ArgumentError: Unknown key: :foreign_key. Valid keys are: :limit, :precision, :scale, :default, :null, :collation, :comment, :primary_key, :if_exists, :if_not_exists, :array, :using, :cast_as, :as, :type, :enum_type, :stored (ArgumentError)
```

As you can see, there is no "foreign_key" valid key. Another reason I have found is the next one, gathered from https://guides.rubyonrails.org/getting_started.html#mvc-and-you:
![image](https://github.com/alexansaa/my_blog_app/assets/60579598/73e1365e-05f1-4b1e-8ec1-0e07ec50856c)